From 1f0fe45e0f9b731a6d651dbe5cf0d1a8a0ae2ab5 Mon Sep 17 00:00:00 2001
From: Arne Coucheron <arco68@gmail.com>
Date: Wed, 6 Dec 2017 00:52:39 +0100
Subject: [PATCH 028/249] serrano: Add custom TWRP links

No idea if this shit works, as there's no documentation or
how-to's anywhere, so had to figure it out myself.

Change-Id: I31e175a2ec1a73a22b734ca82ef32e01bebe4384
---
 _data/devices/serrano3gxx.yml  | 1 +
 _data/devices/serranodsdd.yml  | 1 +
 _data/devices/serranoltexx.yml | 1 +
 3 files changed, 3 insertions(+)

diff --git a/_data/devices/serrano3gxx.yml b/_data/devices/serrano3gxx.yml
index 6dc2dc4..d37748e 100644
--- a/_data/devices/serrano3gxx.yml
+++ b/_data/devices/serrano3gxx.yml
@@ -11,6 +11,7 @@ cpu: Krait
 cpu_cores: '2'
 cpu_freq: 1.7 GHz
 current_branch: 14.1
+custom_twrp_link: http://www.mediafire.com/file/30lqgdzllgi6qy5/recovery.img
 depth: 8.9 mm (0.35 in)
 download_boot: With the device powered off, hold <kbd>Volume Down</kbd> + <kbd>Home</kbd> + <kbd>Power</kbd>.
 gpu: Adreno 305
diff --git a/_data/devices/serranodsdd.yml b/_data/devices/serranodsdd.yml
index b01bd38..2991295 100644
--- a/_data/devices/serranodsdd.yml
+++ b/_data/devices/serranodsdd.yml
@@ -11,6 +11,7 @@ cpu: Krait
 cpu_cores: '2'
 cpu_freq: 1.7 GHz
 current_branch: 14.1
+custom_twrp_link: http://www.mediafire.com/file/4m9u74p8akndor8/recovery.img
 depth: 8.9 mm (0.35 in)
 download_boot: With the device powered off, hold <kbd>Volume Down</kbd> + <kbd>Home</kbd> + <kbd>Power</kbd>.
 gpu: Adreno 305
diff --git a/_data/devices/serranoltexx.yml b/_data/devices/serranoltexx.yml
index a93e129..aae6b55 100644
--- a/_data/devices/serranoltexx.yml
+++ b/_data/devices/serranoltexx.yml
@@ -11,6 +11,7 @@ cpu: Krait
 cpu_cores: '2'
 cpu_freq: 1.7 GHz
 current_branch: 14.1
+custom_twrp_link: http://www.mediafire.com/file/3h5z8jr3r5itk5i/recovery.img
 depth: 8.9 mm (0.35 in)
 download_boot: With the device powered off, hold <kbd>Volume Down</kbd> + <kbd>Home</kbd> + <kbd>Power</kbd>.
 gpu: Adreno 305
-- 
2.17.1

